daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/value/Value.scala

```diff
@@ -388,13 +388,15 @@ object Value extends CidContainer1WithDefaultCidResolver[Value] {
     }
 
     object V1 {
+      val maxSuffixLength = 94
```

Comment:
I'm just wondering: from where does that magic number come from? If you would like to follow the Scala style guide this constant should be `MaxSuffixLength`.